src/Features.php

```diff
@@ -170,6 +172,15 @@ public static function get_fields(): array {
 				'id'   => self::GOOGLE_SHEET_REPLACES_SMARTSHEET,
 				'type' => 'checkbox',
 			],
+			[
+				'name' => __( 'Enable core block patterns', 'planet4-master-theme-backend' ),
+				'desc' => __(
+					'Allows using the default block patterns that come with WordPress.',
+					'planet4-master-theme-backend'
+				),
+				'id'   => self::CORE_BLOCK_PATTERNS,
+				'type' => 'checkbox',
+			],
```

Comment:
Let's wrap this inside a check for `WP_APP_ENV` being `development`. To avoid people enabling that on production. At least till we review them.

Reply:
I also moved 2 other fields (WP template editor and experimental theme editor) into this check, that are definitely not intended to be enabled on production sites at the moment.